ravedude/src/main.rs

```diff
@@ -69,9 +74,9 @@ struct Args {
     /// * trinket-pro
     /// * trinket
     /// * nano168
-    /// * duemilanove
+    /// * duemilanovepriori
```

Comment:
What's this change about?

Reply:
Oops, that's embarrassing. I must've accidentally pasted or typed something and somehow missed it. My bad, it's been removed.